3_javascript/javascript1/week1/homework/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>JavaScript1</title>
+  </head>
+  <body>
+    <script>
+      let yearOfBirth = 1989;
+      let yearFuture = 2027;
+      let age = yearFuture - yearOfBirth;
+
+      const sentence =
+        "You will be " + " " + age + " " + "years old in 2027" + ".";
+
+      console.log(sentence);
+      // console.log("Monika");
```

Comment:
You can as well remove this unused code from the final version :innocent:

Reply:
Oh yes that was me trying to see if anything works :D